muttator/content/mail.js

```diff
@@ -1179,7 +1179,12 @@ const Mail = Module("mail", {
                     services.get("smtpService").defaultServer = server;
                     return value;
                 },
-                completer: function (context) [[s.key, s.serverURI] for ([, s] in Iterator(mail.smtpServers))],
+                completer: function (context) {
+                    return Array.from((function* () {
+                        for (let [, s] in Iterator(mail.smtpServers))
```

Comment:
Sure, `smtpServers` property is Array type, therefore, can simplify to replace with `return mail.smtpServers.map(s => [s.key, s.serverURI]);`